booking-up-for-beauty/Pratster95/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import "time"
+import "fmt"
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	myDate, _ := time.Parse("1/2/2006 15:04:05", date)
+	return myDate
+	panic("Please implement the Schedule function")
```

Comment:
I think this `panic` won't execute since it is after a `return`. You will have to handle the `_` which is the error yield by the function. See the first `Parse` call in this example: https://pkg.go.dev/time#example-Time.Format